pkg/daemon/daemon.go

```diff
@@ -240,6 +245,54 @@ func NewClusterDrivenDaemon(
 	return dn, nil
 }
 
+const (
+	logindNewSessionMessageIDMatch = "MESSAGE_ID=8d45620c1a4348dbb17410da57c60c66"
+)
+
+func (dn *Daemon) detectEarlySSHAccesses() error {
+	journal, err := sdjournal.NewJournal()
+	if err != nil {
+		return err
+	}
+	defer journal.Close()
+	if err := journal.AddMatch(logindNewSessionMessageIDMatch); err != nil {
+		return err
+	}
+	if err := journal.SeekHead(); err != nil {
```

Comment:
Right, but then restarting the node or MCD will always reinstall the annotation, even if it was manually cleared in the past. How about only doing this if we're bootstrapping? (See `getStateAndConfigs()`). Then from that point on, the login monitor will catch subsequent accesses. (Though it still leaves a tiny window during the time the nodes are restarted on upgrades).

Reply:
Fixed this and added another check on node startup after reboot

Reply:
alright, this still doesn't cover the case when someone clears the annotation and you reboot on upgrades right? this will be called _at every boot_, correct?

Reply:
Fixed here https://github.com/openshift/machine-config-operator/pull/381/files#diff-06b532a2a5a5d3ad1d5341b3e00945efR555